components/feature/autofill/src/main/java/mozilla/components/feature/autofill/structure/ParsedStructureBuilder.kt

```diff
@@ -35,10 +40,10 @@ internal class ParsedStructureBuilder<ViewNode, AutofillId>(
         }
     }
 
-    private fun findAutofillIds(rootNode: ViewNode?): Pair<AutofillId?, AutofillId?> =
+    private fun findAutofillIds(rootNode: ViewNode?): Pair<Pair<AutofillId?, String?>?, Pair<AutofillId?, String?>?> =
```

Comment:
Heh, at this point this structure is so complex that I think we should probably introduce a small `data class` that can hold this data and give us nice properties to access - instead of unpacking multi-level pairs :)

Reply:
Fixed